src/main/java/bio/terra/app/controller/CohortBuilderApiController.java

```diff
@@ -0,0 +1,432 @@
+package bio.terra.app.controller;
+
+import bio.terra.controller.CohortBuilderApi;
+import bio.terra.model.Attribute;
+import bio.terra.model.Cohort;
+import bio.terra.model.CohortCreateInfo;
+import bio.terra.model.CohortList;
+import bio.terra.model.CohortUpdateInfo;
+import bio.terra.model.ConceptSet;
+import bio.terra.model.ConceptSetCreateInfo;
+import bio.terra.model.ConceptSetList;
+import bio.terra.model.ConceptSetUpdateInfo;
+import bio.terra.model.CountQuery;
+import bio.terra.model.Criteria;
+import bio.terra.model.CriteriaGroup;
+import bio.terra.model.DataType;
+import bio.terra.model.DisplayHint;
+import bio.terra.model.DisplayHintDisplayHint;
+import bio.terra.model.DisplayHintEnum;
+import bio.terra.model.DisplayHintEnumEnumHintValues;
+import bio.terra.model.DisplayHintList;
+import bio.terra.model.DisplayHintNumericRange;
+import bio.terra.model.HintQuery;
+import bio.terra.model.Instance;
+import bio.terra.model.InstanceCount;
+import bio.terra.model.InstanceCountList;
+import bio.terra.model.InstanceHierarchyFields;
+import bio.terra.model.InstanceList;
+import bio.terra.model.InstanceRelationshipFields;
+import bio.terra.model.Literal;
+import bio.terra.model.LiteralValueUnion;
+import bio.terra.model.Query;
+import bio.terra.model.ValueDisplay;
+import io.swagger.annotations.Api;
+import java.time.OffsetDateTime;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+@Api(tags = {"cohort-builder"})
+public class CohortBuilderApiController implements CohortBuilderApi {
+
+  @Autowired
+  public CohortBuilderApiController() {}
+
+  @Override
+  public ResponseEntity<CohortList> listCohorts(
+      @RequestParam(value = "offset", required = false, defaultValue = "0") Integer offset,
+      @RequestParam(value = "limit", required = false, defaultValue = "10") Integer limit) {
```

Comment:
In general you don't want to copy down swagger generated annotation to subclasses, as any yaml changes you make won't be applied until the new generated content is copied to the subclass. So having the annotations here is redundant and can lead to unexpected problems.
```suggestion
  public ResponseEntity<CohortList> listCohorts(Integer offset,Integer limit) {
```